helm/frost-server/values.yaml

```diff
@@ -219,8 +223,26 @@ frost:
       mqtt:
         message_size: 8092
 
+    ingress:
+      enabled: false      
+      ## Set a specific ingress-class for the ingress ([default] not set)
+      # className: azure-application-gateway
+      ## Defines the type of ingress-Controller in use (nginx [default], agic or traefik) 
+      # ingressProvider: nginx 
+      ## Additional annotations
+      annotations:
+        nginx.ingress.kubernetes.io/proxy-read-timeout: "3600"
+        nginx.ingress.kubernetes.io/proxy-send-timeout: "3600"
+        # cert-manager.io/cluster-issuer: letsencrypt
+        # cert-manager.io/issue-temporary-certificate: "true"
+      tls:
+        enabled: true
```

Comment:
Default should be set to `false`, since the TLS-Secret is not part of the HELM-Chart.